lte/gateway/deploy/roles/ovs_deploy/tasks/main.yml

```diff
@@ -11,69 +11,112 @@
 # limitations under the License.
 
 
-- fail: msg="PACKAGE_LOCATION is required"
-  when:
-    - PACKAGE_LOCATION is not defined
-
-- name: Include vars of all.yaml.
+- name: Include vars of all.yaml
   include_vars:
     file: all.yaml
-    name: all_vars
 
-- name: Converting PACKAGE_LOCATION to absolute path
-  shell: realpath {{ PACKAGE_LOCATION }}
-  register: PACKAGE_LOCATION_ABSOLUTE
+- name: Configuring private package manager.
+  copy:
+    content: >
+      deb {{ ovs_pkgrepo_proto }}://{{ ovs_pkgrepo_host }}{{ ovs_pkgrepo_path}}
+      {{ ovs_pkgrepo_dist }} {{ ovs_pkgrepo_component }}
+    dest: /etc/apt/sources.list.d/{{ ovs_pkgrepo_name }}.list
+  become: yes
+
+- name: Add unvalidated Apt signing key.
```

Comment:
let's just remove this whole block and just do how we did for packages.magma.etagecom.io

Reply:
this was essential for debugging bare metal install with custom local repo... removing this will mean testing is only possible against official repo